### DISCUSSION TITLE
Another relation (expand) query!

### DISCUSSION THREAD

**Discussion by steve-potts** [pocketbase/pocketbase#2666] [Q&A] [answered]
Really struggling to get pocketbase relation and expand!
looking at this imagine would someone help with how to expand , 

so, a booking has a coop and a customerid
and customers will have an address and email address (or many)

![image](https://github.com/pocketbase/pocketbase/assets/4457038/32a03778-ddac-419d-b4be-c321b50005bc)

im currently using angular/ service so for example, im trying things like..
  getCustomers() {
    return this.http.get(`${environment.apiUrl}/collections/bookings/records?expand=customers(customer_id),coops(coop_id)`);
  }

thanks if anyone can help :)

**Comment by ganigeorgiev** [comment, marked as answer]
The parenthesis expand syntax is for indirect `relation` fields.
From the screenshot that doesn't seem the case as `custom_id` and `coop_id` are direct `relation` fields in the `bookings` collection. OR in other words, try with:

```js
return this.http.get(`${environment.apiUrl}/collections/bookings/records?expand=customer_id,coop_id`);
```

If you haven't already, you can also check https://pocketbase.io/docs/expanding-relations/.

**Reply by steve-potts** [reply]
i read it but obviously misunderstood , your right, you are a legend
